### PR TITLE
UWM-RAR: RAR Search 577585248

### PR DESCRIPTION
**Pull request by blaquehat**
# Description:
Create a results page that works with the Search Box in the Header. This contains configuration to setup Search API and Views Page.

### CODE REVIEW

**Review by sodacrackers — APPROVED** [review]
Could this also require an update to `config/rar/core.extension.yml`? 
On an cim I see a few Search API messages such as _Configuration search_api.index.content depends on the Search API module that will not be installed after import._

File: config/rar/views.view.search.yml [hunk 1]
```diff
@@ -0,0 +1,293 @@
...
+          id: area
+          table: views
+          field: area
+          relationship: none
+          group_type: group
+          admin_label: ''
+          empty: true
+          tokenize: false
+          content:
+            value: "We are sorry your search did not return any results. \r\n\r\nBut, we’re open to suggestions! Please take a moment and <a href=\"/contactus\">send us your idea</a> for a new article.\r\n"
```
[the remark this answers is not in the document]
Comment (blaquehat) [reply config/rar/views.view.search.yml:227]: @sodacrackers, Double checked the test site and contact us page is http://rar.uwmedicinetest.org/contactus. The current live site is using https://rightasrain.uwmedicine.org/contactus.

[the remark this answers is not in the document]
Comment (blaquehat) [reply config/rar/views.view.search.yml:227]: @sodacrackers, I updated the core.extension.yml.

**Comment by blaquehat** [comment]
Closing because too many changes that need to be re-approved.

**Pull request closed without merging**